src/Modules/score.js

```diff
@@ -1,9 +1,15 @@
-const { v4: uuidv4 } = require('uuid');
+async function userScores(a, b) {
```

Comment:
- [ ] please use an arrow function instead as they will help introduce concise body syntax, and implicit return. nice work

you can do it like this 
```
const userScore = async () => {
   // do something
}
```